apps/native/app/src/screens/wallet-pass/wallet-pass.tsx

```diff
@@ -161,27 +199,47 @@ export const WalletPassScreen: NavigationFunctionComponent<{
     fetchPolicy: 'network-only',
     variables: {
       input: {
-        licenseType: item?.license.type ?? id,
+        licenseType: type,
+        licenseId: id,
```

Comment:
The only licenses that support getting the licenseId here as a parameter are Passports, Identity Documents and Ehic cards, otherwise the endpoint wants just the type and then either no licenseId or 'default'.